zhaquirks/smartwings/wm25lz.py

```diff
@@ -0,0 +1,105 @@
+"""Device handler for Smartwings blinds."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+)
+
+from zhaquirks import DoublingPowerConfigurationCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class InvertedWindowCoveringCluster(CustomCluster, WindowCovering):
+    """WindowCovering cluster implementation.
+
+    This implementation inverts the reported covering percent for non standard
+    devices that don't follow the reporting spec.
+    """
+
+    cluster_id = WindowCovering.cluster_id
+    CURRENT_POSITION_LIFT_PERCENTAGE = 0x0008
+    CMD_GO_TO_LIFT_PERCENTAGE = 0x0005
+
+    def _update_attribute(self, attrid, value):
+        if attrid == self.CURRENT_POSITION_LIFT_PERCENTAGE:
+            value = 100 - value
+        super()._update_attribute(attrid, value)
+
+    async def command(
+        self, command_id, *args, manufacturer=None, expect_reply=True, tsn=None
+    ):
```

Comment:
Add the `**kwargs` to the method:
```python
    async def command(
        self, command_id, *args, manufacturer=None, expect_reply=True, tsn=None, **kwargs
    ):
```

* https://github.com/zigpy/zigpy/blob/be5b2914a12ac6a19af80c5a83dab40d7957aa01/zigpy/quirks/__init__.py#L163-L172